Cargo.toml

```diff
@@ -21,7 +21,7 @@ stream = []
 log = "0.4"
 futures = "0.3"
 pin-project = "0.4"
-tokio = "0.2"
+tokio = { version = "0.2", default-features = false, features = ["io-util"] }
```

Comment:
Nit, but tokio's Cargo.toml has:
```
# Include nothing by default
default = []
```
so `default-features = false` is a no-op.

Reply:
Well that's assuming they won't add some. But I guess that's somewhat unlikely.